updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -8,13 +8,13 @@ toc::[]
 
 Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized.
 You must update from {product-title} 4.8 to 4.9 and then to 4.10. You cannot update from {product-title} 4.8 to 4.10 directly.
-However, beginning with the update from {product-title} 4.8 to 4.9 to 4.10, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-master hosts.
+However, beginning with the update from {product-title} 4.8 to 4.9 to 4.10, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-master hosts. For more information, see link:https://access.redhat.com/labs/ocpupgradegraph/update_path[Red Hat OpenShift Container Platform Update Graph].
```

Comment:
IMO, https://access.redhat.com/labs/ocpupgradegraph/update_path may be an useful tool in some scenarios. But it's not ocp features and so QE don't test it. So I'm afraid it's a risk to provide such link for customers in offical ocp doc.

Reply:
This link "https://access.redhat.com/labs/ocpupgradegraph/update_path" already exists in section 9.1.4
It is linked as "[Red Hat OpenShift Container Platform Upgrade Graph visualizer and update planner](https://access.redhat.com/labs/ocpupgradegraph/update_channel)"
IMO it should be included.

Reply:
@[julienlim](https://github.com/julienlim) ^